CMakeLists.txt

```diff
@@ -102,17 +100,21 @@ ffig_add_library(NAME Tree INPUTS tests/input/Tree.h NOEXCEPT PYTHON CPP D SWIFT
 ffig_add_library(NAME Asset INPUTS tests/input/Asset.h PYTHON LUA)
 ffig_add_library(NAME Animal INPUTS tests/input/Animal.h PYTHON)
 
-# build dotnet test dll
-# FIXME: Work out how to get dependencies calculated more robustly
 if(dotnet_FOUND)
-  file(GLOB TEST_INPUT ${CMAKE_SOURCE_DIR}/tests/input/*.h)
-  add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/generated/ffig.net.dll
-    COMMAND dotnet build ffig.net.csproj -o .
-    DEPENDS ${CMAKE_BINARY_DIR}/generated/Shape.cs;${CMAKE_BINARY_DIR}/generated/Tree.cs
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/generated
-    COMMENT "Building DOTNET assembly ffig.net.dll")
-  add_custom_target(ffig.net ALL
-    DEPENDS ${CMAKE_BINARY_DIR}/generated/ffig.net.dll)
+  include(dotnet)
+  add_dotnet_library(NAME ffig.net 
+    DIRECTORY ${CMAKE_BINARY_DIR}/generated
```

Comment:
Do you really need to specify the output folder?

Reply:
DIRECTORY specifies where the project file will be found.

I'm not sure if that's what you mean.